packages/eslint-plugin/tests/rules/no-unsafe-assignment.test.ts

```diff
@@ -269,78 +268,84 @@ const x: Set<Set<Set<string>>> = new Set<Set<Set<any>>>();
         },
       ],
     },
-    ...batchedSingleLineTests({
+
+    {
       code: `
 const x = [...(1 as any)];
+      `,
+      errors: [{ messageId: 'unsafeArraySpread' }],
+    },
+    {
+      code: `
 const x = [...([] as any[])];
       `,
-      errors: [
-        {
-          messageId: 'unsafeArraySpread',
-          line: 2,
-          column: 12,
-          endColumn: 25,
-        },
-        {
-          messageId: 'unsafeArraySpread',
-          line: 3,
-          column: 12,
-          endColumn: 28,
-        },
-      ],
```

Comment:
is there a reason why we are not preserving loine/col/endCol ? 

its a good practice to add those to validate if error message is reported in correct place

Reply:
because I couldn't be bothered remapping each one before and after the prettier format.
it already took long enough to extract those ones as it is.

soon™️ I'll implement snapshot testing in the fork which will snapshot an error underline and will make the line/col assertions unnecessary cruft anyways.